.github/workflows/release-management.yml

```
@@ -0,0 +1,28 @@
+    steps:
+      - uses: actions/checkout@v4
+      - name: Read changelog for this version
+        id: changelog
+        run: |
+          VERSION=${GITHUB_REF_NAME}
+          NOTES=$(awk '/^## /{p=($2==VERSION);next} p{print}' VERSION=$VERSION CHANGELOG.md || true)
+          echo "notes<<EOF" >> $GITHUB_OUTPUT
+          echo "$NOTES" >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
+      - name: Create release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)